tests/chainer_tests/training_tests/extensions_tests/test_print_report.py

```diff
@@ -0,0 +1,45 @@
+import sys
+import unittest
+
+from chainer import testing
+from chainer.training import extensions
+from mock import MagicMock
```

Comment:
Please separate `mock` from `chainer` imports, and please avoid directly importing a class (`MagicMock`).
Please see `tests/chainer_tests/test_variable.py`, for example.